tasks/eth/zora-002-fp-upgrade/.env

```diff
@@ -0,0 +1,28 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+
+OWNER_SAFE=0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A
+COUNCIL_SAFE=0xc2819DC788505Aac350142A7A707BF9D03E3Bd03
+FOUNDATION_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
+LIVENESS_GUARD=0x24424336F04440b1c28685a38303aC33C9D14a25
+
+# No SAFE_NONCE_* values are specified, we just use the current nonces
+# SAFE_NONCE_0X5A0AAE59D09FCCBDDB6C6CCEB07B7279367C3D2A=
+# SAFE_NONCE_0XC2819DC788505AAC350142A7A707BF9D03E3BD03=
+# SAFE_NONCE_0X847B5C174615B1B7FDF770882256E2D3E95B9D92=
+
+# Expected prestate
+EXPECTED_PRESTATE=0x0354eee87a1775d96afee8977ef6d5d6bd3612b256170952a01bf1051610ee01
```

Comment:
Proofs team to validate this prestate works for zora

Reply:
I believe it doesn't matter but will confirm

Reply:
The prestate should work for zora. It's compatible with holocene-configured zora chain.

Reply:
Ok great, resolving this thread